src/zino/stateconverter/flapping_converter.py

```diff
@@ -20,34 +22,43 @@ def set_flapping_state(old_state: OldState, new_state: ZinoState):
         _set_flaps(linedata, new_state)
     for linedata in old_state["::lastFlap"]:
         _set_last_flap(linedata, new_state)
-    for linedata in old_state["::lastFlap"]:
+    for linedata in old_state["::lastAge"]:
         _set_last_age(linedata, new_state)
 
 
 def _set_first_flap(linedata: LineData, state: ZinoState):
-    _log.info("firstFlap is not supported")
+    timestamp = datetime.fromtimestamp(int(linedata.value), tz=timezone.utc)
+    _get_flapping_state_for(linedata, state).first_flap = timestamp
 
 
 def _set_flap_hist_val(linedata: LineData, state: ZinoState):
-    _log.info("flapHistVal is not supported")
+    _get_flapping_state_for(linedata, state).hist_val = float(linedata.value)
 
 
 def _set_flapped_above_threshold(linedata: LineData, state: ZinoState):
-    _log.info("flappedAboveThreshold is not supported")
+    _get_flapping_state_for(linedata, state).last_age = bool(int(linedata.value))
```

Comment:
I assume this is a mistake, setting `last_age` in func for `flapped_above_threshold`